api/internal.json

```diff
@@ -230,6 +230,34 @@
       "set_code": "ELD",
       "reason": "Banned for introducing too many risks and design constraints to the future of Standard.",
       "announcement_url": "https://magic.wizards.com/en/articles/archive/news/june-1-2020-banned-and-restricted-announcement"
+    },
+    {
+      "card_name": "Wilderness Reclamation",
+      "card_image_url": "https://img.scryfall.com/cards/png/front/5/4/54af08f7-9c6c-464e-b2f7-2b5803f36481.png",
+      "set_code": "RNA",
+      "reason": "Banned for its dominance in high level play and to reduce the metagame share of ramp decks in general.",
+      "announcement_url": "https://magic.wizards.com/en/articles/archive/news/august-8-2020-banned-and-restricted-announcement"
```

Comment:
Weirdly enough, the URL they used is `august-8`, and replacing it with `august-3` doesn't work. Just a far warning this URL might break if they update it on their side eventually.